app/__init__.py

```diff
@@ -2,12 +2,12 @@
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-from flask_cors import CORS, cross_origin
+#from flask_cors import CORS, cross_origin
```

Comment:
Don't commit these things as commented, leave them uncommented in your local if that's how it works for you